Chainlit/chainlit
# Add an option to disable the prompt playground

### Changes 

- The playground display behavior can be handled through the chainlit config (TODO: Update documentation)
- The `prompt` data is filtered from response
- Updated `.gitignore` to allow e2e config tracking

### How to test?

- Run `chainlit run ../cypress/e2e/hide_prompt_playground/main.py` 
- Notice that we are not displaying the playground
